modules/tensor_mechanics/test/tests/torque_reaction/tests

```diff
@@ -1,34 +1,53 @@
 [Tests]
   [./torque_reaction_2D]
+    design = TorqueReaction.md
+    issues = '#6484'
+    requirement = 'The TorqueReaction Postprocessor shall compute the torque for an applied deformation at the end of a beam in 2D'
     type = 'Exodiff'
     input = 'torque_reaction_tm.i'
     exodiff = 'torque_reaction_tm_out.e'
   [../]
   [./torque_reaction_3D]
+    design = TorqueReaction.md
+    issues = '#6484'
+    requirement = 'The TorqueReaction Postprocessor shall compute the torque for an applied deformation at the end of a beam in 3D'
     type = 'Exodiff'
     input = 'torque_reaction_3D_tm.i'
     exodiff = 'torque_reaction_3D_tm_out.e'
     abs_zero = 1.e-8
   [../]
   [./torque_reaction_cylinder]
+    design = 'TorqueReaction.md DisplacementAboutAxis.md'
```

Comment:
Any chance you'd be willing to rename the input/gold file pairs from the previous two tests to remove the `_tm`?

Reply:
I thought the same thing, and was just being lazy.  Sure, I'll fix that. Then you're going to want me to convert them all to use the Action too...